### DISCUSSION TITLE
Running HyperHDR on an LG WebOS 22 (SDK 7.3.1) TV via Developer Mode?

### DISCUSSION THREAD

**Discussion by PlasmaSoftUK** [awawa-dev/HyperHDR#652] [General]
Hi I was looking for Discord channel but came across this discussion page, so here I am.

I have a new TV which is an LG 75QNED916QE, as I understand it I'm not able to root my TV because its too new, however I have enabled developer mode and got the SDK CLI on my MacBook and managed to connect the two together ..

`plasma@Plasmas-MBP % ares-device-info --device tv
modelName : 75QNED916QE
sdkVersion : 7.3.1
firmwareVersion : 03.33.60`

I'm wanting to install HyperHDR so that I can run an AmbiLight setup, is it simply a matter of grabbing the latest release of 'hyperhdr-webod-loader' and installing it using ...

`ares-install --device tv ./org.webosbrew.hyperhdr.loader_0.1.6_all.ipk`

Does this install all the necessary capture dependencies too? Or do I need to do something fancy to get the app to run as root? There is almost no information on the GitHub page for it, other than the recommended install is via HomeBrew Channel. I'm reasonably technical and should be able to follow the briefest of instructions. I've also seen in the search results here that awawa-dev doesn't maintain webos repos, so maybe this isn't the right place to ask, but is anyone able to offer any advice / a page with a guide or a list of what I need to do?

Many Thanks!

**Comment by awawa-dev** [comment]
Hi
I think you should first check on their discord whether piccap specifically requires root and developer mode is insufficient. HyperHDR does not have any built-in components that work directly with webos capturing.

**Reply by PlasmaSoftUK** [reply]
Ok thanks for the quick reply, I've just found a Gist that looks like it will help me root the TV via the developer mode, so then I should be able to install them as normal via the Homebrew Channel. For anyone else looking to do a similar thing have a read here, there is also a link to their Discord ...

[https://gist.github.com/throwaway96/e811b0f7cc2a705a5a476a8dfa45e09f](url)
